docs/notification_system.md

```diff
@@ -99,6 +100,26 @@ Hipchat allows you to create a team with limited users and message history for f
 
 https://www.hipchat.com/server
 
+## Mattermost
+
+The mattermost notification integration uses Incoming Webhooks. These must be enabled in the System Console of Mattermost. If the user wishes to allow Ansible Tower notifications to modify the Icon URL and username of the notification then they must enabled these options as well.
```

Comment:
My one complaint is that we should provide a little bit more information on how to actually turn this on. It wasn't immediately obvious to me where to turn on integrations and then where to find the webhook integrations.  Just a little bit more here would probably do well.

Might also point out that a password isn't required because the notification webhook url itself is the secret.

Nicely done, though... I'm impressed with host fast you turned this around.

Reply:
Thanks!

I can definitely beef up the docs a bit, as well as apply the other changes that were pointed out. I'll push up the changes as soon as I have a bit of free time today.